edge/pkg/edged/edged.go

```diff
@@ -1015,19 +1015,17 @@ func (e *edged) podRemoveWorkerRun(consumers int) {
 
 func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	podName := namespacedName.Name
-	klog.Infof("start to consume added pod [%s]", podName)
+	klog.V(4).Infof("start to consume added pod [%s]", podName)
 	pod, ok := e.podManager.GetPodByName(namespacedName.Namespace, podName)
 	if !ok || pod.DeletionTimestamp != nil {
 		return apis.ErrPodNotFound
 	}
 
 	if err := e.makePodDataDirs(pod); err != nil {
-		klog.Errorf("Unable to make pod data directories for pod %q: %v", format.Pod(pod), err)
```

Comment:
Do we need to return all the info in `klog.Errorf`?

Reply:
ok